project/Release.scala

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka
+
+import sbt.Keys._
+import sbt._
+
+object Release {
```

Comment:
this is probably very obsolete, when it's time for adding releases you can take a look at r2dbc build (and github workflows)

Reply:
Deleted. I'll keep that that in mind